proxy/certs/certs.go

```diff
@@ -186,10 +196,17 @@ func (s *RemoteCertSource) Local(instance string) (ret tls.Certificate, err erro
 	createEphemeralRequest := sqladmin.SslCertsCreateEphemeralRequest{
 		PublicKey: pubKey,
 	}
+	// If IAM login is enabled, add the OAuth2 token into the ephemeral
+	// certificate request, always refreshing the token to ensure its expiration
+	// is far enough in the future.
```

Comment:
nit: I'd suggest separating the second part of this sentence and move it down to L207. (it doesn't seem relevant to the reader unless they go down that path)

Reply:
Sounds good.